docs/user-guides/contributors_guide.md

```diff
@@ -0,0 +1,71 @@
+- Discussing the current state of the code
+- Submitting a fix
+- Proposing new features
+- Becoming a maintainer
+
+## We Develop with Github
+We use github to host code, to track issues and feature requests, as well as accept pull requests.
+
+## We Use [Github Flow](https://guides.github.com/introduction/flow/index.html), So All Code Changes Happen Through Pull Requests
+Pull requests are the best way to propose changes to the codebase (we use [Github Flow](https://guides.github.com/introduction/flow/index.html)). We actively welcome your pull requests:
+
+1. Fork the repo and create your branch from `master`.
+2. If you've added code that should be tested, add tests.
+3. Ensure the test suite passes.
+4. Make sure your code lints.
+5. Issue that pull request!
+
+## Report bugs using Github's [issues](https://github.com/hyperledgerkochi/KOBman/issues)
+We use GitHub issues to track public bugs. Report a bug by [opening a new issue](https://github.com/hyperledgerkochi/KOBman/issues); it's that easy!
+
+
+### Bugs and New Features
+
+We keep a distinction between New Features and Bugs/Issues. We try to minimise the noise in our Github Issues stream by first having a conversation on approprate media (eg:  [KochiOrgBook! Whatsapp Group](https://chat.whatsapp.com/El6d3aAc6pYLEDEdQPVABY). ) 
+
+- User Issues can be raised in our [issues section](https://github.com/hyperledgerkochi/KOBman/issues).
+- New Features or Enhancements can be discussed in our issue section.
+
+After accepting a new feature or confirming that a bug was found, an Issue may be raised filling in the provided issue template on the KOBman issue tracker.
+
+**Unfortunately we will close any Github Issues that have not followed this process of prior discussion.**
+
+## Write bug reports with detail, background, and sample code
+[This is an example](https://github.com/asa1997/KOBman/issues/8) of a bug reported by a contributor .
+
+**Great Bug Reports** tend to have:
+
+- A quick summary and/or background
+- Steps to reproduce
+  - Be specific!
+  - Give sample code snapshots if you can. [This contributor sample issue](https://github.com/asa1997/KOBman/issues/33) includes sample code snapshots that *anyone* can understand
+- What you expected would happen
+- What actually happens
+- Notes (possibly including why you think this might be happening, or stuff you tried that didn't work)
+
+People *love* thorough bug reports. 
+
+
+## Processes expected to follow by each contributors:
+
+- [Process for pull request](https://github.com/hyperledgerkochi/KOBman/blob/master/docs/user-guides/process-to-raise-a-pull-request.md)
+
+- [Process for issues](https://github.com/hyperledgerkochi/KOBman/blob/master/docs/user-guides/process-to-raise-an-issue.md)
+
+
+
+## Use a Consistent Coding Style
+It is recommended to follow the [KOBman Checklist](https://github.com/hyperledgerkochi/KOBman/blob/master/docs/KOBman%20Script%20Review%20Checklist.md) before submitting any pull requests.

```

Comment:
You must follow the coding standard described in the KOBman review checklist.

Reply:
@asa1997  , the wording has been updated like suggested